CIP-0XXX/README.md

```diff
@@ -0,0 +1,540 @@
+---
+CIP: ?
+Title: Cardano Decentralized Message Queue
+Category: ?
+Status: Proposed
+Authors:
+    - Jean-Philippe Raynaud <jp.raynaud@gmail.com>
+    - Arnaud Bailly <arnaud.bailly@iohk.io>
+    - Marcin Szamotulski <marcin.szamotulski@iohk.io>
+    - Armando Santos <armando.santos@iohk.io>
+    - Neil Davies <neil.davies@iohk.io>
+    - Sebastian Nagel <sebastian.nagel@ncoding.at>
+    - ?
+Implementors: 
+    - Cardano Scaling team <https://github.com/cardano-scaling>
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/?
+Created: 2024-08-02
+License: Apache-2.0
+---
+
+# Cardano Decentralized Message Queue
```

Comment:
```suggestion
```
the new template is held at https://github.com/cardano-foundation/CIPs/blob/master/.github/CIP-TEMPLATE.md

please follow

Reply:
@jpraynaud there are several suggestions, with many examples, for updating the old to the new format here (you are using kind of a mix of the old & new formats):
- #389 

If you have difficulty getting started with that, just post here saying so and I'll contribute a review with the necessary formatting changes.